appendices/VK_FOOL_printed_surface.txt

```diff
@@ -0,0 +1,206 @@
+
+include::{generated}/interfaces/VK_FOOL_printed_surface.txt[]
+
+=== Issues
+
+Tons of them, please curb your enthusiasm.
+
+
+1) What should this extension be named?
+
+*RESOLVED*: VK_*_printed_surface, to ensure that we don not box ourselves in on
+            the medium that the swapchain images are being printed onto.
+
+
+2) Should this extension be made platform specific by introducing a dependency on CUPS?
+
+*RESOLVED*: No, for too long printers have been overlooked in graphics APIs
+            without a clear method of standardization. VK_FOOL_printed_surface
+            aims to close this massive gap in the ecosystem by providing the first
+            extension to bring a standardized cross-platform printing API to
+            the Vulkan specification.
+
+
+3) Fax machines share similar base functionality with printers. Will they be supported by this extension?
+
+*RESOLVED*: Yes, although only for their printing capacity. Implementations
+            may: choose to provide out-of-API options for setting up fax-based
+            communication, or help to specify an additional extension
+            introducing more robust Fax support.
+
+
+4) Printers are inherently complex devices which are not yet fully understood
+   by humanity, does the current set of error codes available in Vulkan allow
+   for implementations to adequately express the various states of disarray
+   encountered when a print is not successful?
+
+*RESOLVED*: To the best of our ability we have introduced new Vulkan error codes
+            which should help implementations express the complex state of printers.
+            This list is likely not exhaustive and may: be expanded in a future extension.
+
+
+5) Will this extension support printers with limited character set support,
+   such as teleprinters?
+
+*RESOLVED*: This is not for the specification to declare. The finer print for this
+            answer is left up to the implementation.
+
+6) Typicaly swapchain image count requirements are subject to presentation
+   hardware and are queried at initialization-time. Will this conflict with the
+   reality of dwindling page count?
+
+*RESOLVED*: Page count is a chief concern here, several additional error codes
+            were added to this extension in an effort to address such serious
+            concerns. Page count must: be considered at surface creation time.
+
+
+7) Why would someone want to see a printed frame?
+
+*RESOLVED*: This is a stupid question. For thousands of years humanity has
+            pursued artistic passion and looked to immortalize their creations.
+            This pursuit obviously extends to the printed frame.
+
+
+8) Pre-existing surface implementations would account for surfaceFormat which
+   is confusingly names as it actually represnts the surface color format,
+   rather than the medium of the presentation surface itself. Will this
+   extesnion carry that precedent in light of mulitple supported types of
```

Comment:
I fear that the multiple typos in this non-normative appendix make this extension completely impractical, and thus it must be rejected at present. But please do resubmit after 12 months, if you can work out these problems.